lib/FusionInventory/Agent/Task/Inventory/Win32/Firewall.pm

```diff
@@ -105,7 +105,7 @@ sub _makeProfileAndConnectionsAssociation {
     return unless $DNSRegisteredAdapters;
 
     foreach my $interface (getInterfaces()) {
-        next if ($interface->{STATUS} ne 'Up');
+        next if ($interface->{STATUS} ne 'Up') || !defined($interface->{GUID});
```

Comment:
Okay, so you prefer to skip the interface, but can you confirm the `_retrieveFirewallProfileWithoutDomain()` won't return any profile at line 119 so with finally reach a next at line 126 in the case `$interface->{GUID}` is not defined ?